tests/summarizer/test_image_description_retreiver.py

```diff
@@ -11,23 +11,31 @@ def mocked_requests_post(*args, **kwargs):
     json_data = json.loads(kwargs["data"])
```

Comment:
Add tests for checking if default colour is returned when "imagePropertiesAnnotation" is not present. Think of other corner cases and error scenarios and tests for them as well eg: Image with caption is pruned as expected

Reply:
updated.

Reply:
the test case for image with caption is pruned is present for the "Sundar Pichai" image url

Reply:
updated.

Reply:
Can't find a test that checks whether the image was removed. Could you add a separate test? Or rename an existing test accordingly?

Reply:
Ah i see. Apologies there was error in understanding.
Will amend this accordingly!